src/stravalib/model.py

```diff
@@ -880,33 +777,22 @@ class AthleteSegmentStats(
     pr_elapsed_time: Optional[timedelta] = None
     pr_date: Optional[date] = None
 
-    _field_conversions = {
-        "distance": uh.meters,
-    }
-    elapsed_time: Optional[timedelta] = None  # type: ignore[assignment]
-
     _naive_local = field_validator("start_date_local")(naive_datetime)
 
 
 class AthletePrEffort(
     SummaryPRSegmentEffort,
-    BackwardCompatibilityMixin,
 ):
     # Undocumented attributes:
     distance: Optional[float] = None
     start_date: Optional[datetime] = None
     start_date_local: Optional[datetime] = None
     is_kom: Optional[bool] = None
 
-    _field_conversions = {
-        "distance": uh.meters,
-    }
-    pr_elapsed_time: Optional[timedelta] = None  # type: ignore[assignment]
-
     _naive_local = field_validator("start_date_local")(naive_datetime)
 
     @property
-    def elapsed_time(self) -> Optional[timedelta]:
+    def elapsed_time(self) -> Optional[int]:
```

Comment:
Since we don't use the overridden attribute anymore, this return type has changed.